core/txpool/bundlepool/bundlepool.go

```diff
@@ -73,27 +73,54 @@ type BundlePool struct {
 	config Config
 
 	bundles    map[common.Hash]*types.Bundle
-	bundleHeap BundleHeap
+	bundleHeap BundleHeap // least price heap
 	mu         sync.RWMutex
 
 	slots uint64 // Number of slots currently allocated
 
-	simulator BundleSimulator
+	// bundleMetrics for mev data analyst
+	bundleMetrics   map[int64][]*types.Bundle // receivedBlockNumber -> bundle
```

Comment:
as long as this is just a metrics, do we need store the whole bundle?
A bundle is like :
```
type Bundle struct {
	Txs               Transactions
         ...
	Price *big.Int // for bundle compare and prune

	// caches
	hash atomic.Value
	size atomic.Value
}
```
Transactions will cost too much memory, I think txhashes is good enough?

Reply:
fixed